Dockerfile

```diff
@@ -1,14 +1,20 @@
 FROM nginx:1.10
 MAINTAINER Wyatt Pearsall<Wyatt.Pearsall@cfpb.gov>
 
-RUN mkdir -p /usr/src/app
+RUN apt-get update && \
```

Comment:
I think the short answer for now is _yes_.  We _could_ have a separate `Dockerfile.nginxonly` or perhaps get clever with buildargs, but I'd rather keep the setup simple for now.